04-rendering-output.Rmd

```diff
@@ -107,6 +107,28 @@ At a lower level, the arguments inside the parentheses will be used as the argum
 
 While this approach can be used to statically define the size of an image, it will not work for dynamic sizing of an image. If you wish to dynamically size images, you will need render and capture the graphical output yourself and return the contents with the appropriate `Content-Type` header. See the existing image renderers as a model of how to do this.
 
+### Return a file
+
+Sometimes it is good to have an endpoint where a particular file lives. This could be a [mounted](#mount-static) directory where serveral files would live and querying any of these would "save" the file at the front end. However, this could be an R object that is generated on the fly and returned at a particular URL. For example, `http://localhost:8000/docs/users.txt`. This could be achieved by: 
```

Comment:
"several" is still misspelled. 

I don't understand `and querying any of these would "save" the file at the front end`. Maybe we just end the sentence at "live"?